core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -975,6 +977,8 @@ public ServiceError getResponseCode() {
 				};
 				throw new ServiceResponseException(new ServiceResponse(soapFaultDetails));
 			}
+			log.debug("Mail message [{}] was received by mail address [{}]; received-representing: {}",
+				emailMessage.getId(), receivedBy.getAddress(), receivedRepresenting.getAddress());
```

Comment:
Kan dit hier? 
```suggestion
			log.debug("Mail message [{}] was received by mail address [{}]; received-representing: {}",
				emailMessage::getId, receivedBy::getAddress, receivedRepresenting::getAddress);
```

Reply:
Nee, om 2 redenen niet.

1. Deze method gaat weg met de hierna aankomende changes. Dit nu aanpassen in een backport-PR gaat conflicts geven
2. Belangrijker nog: al deze methods throwen een checked exception en dat kan niet vanuit de `Supplier`. (Daarom dat het framework een `ThrowingSupplier` heeft, maar die kan hier niet worden gebruikt). Dus wat jij voorstelt had ik al geprobeerd, maar gaat syntax fouten geven in Java.